<launch>
   <arg name="traj_drone1" default="$(find bridge_px4)/trajectories/square_relative.csv"/>
   <arg name="traj_drone2" default="$(find bridge_px4)/trajectories/square_relative.csv"/>
   <arg name="traj_drone3" default="$(find bridge_px4)/trajectories/square_relative.csv"/>

   <arg name="t_fs" default="60"/>
   <arg name="err_tol" default="0.225"/>

   <group ns="drone1">
   	<node pkg="bridge_px4" type="sp_pos_e_node" name ="sp_pos_e_node" output="screen">
           <param name="t_fs" value="$(arg t_fs)"/>
	   <param name="err_tol" value="$(arg err_tol)"/>
	   <param name="traj_id" value="$(arg traj_drone1)"/>
	</node>
   </group>

   <group ns="drone2">
   	<node pkg="bridge_px4" type="sp_pos_e_node" name ="sp_pos_e_node" output="screen">
        <param name="t_fs" value="$(arg t_fs)"/>
	<param name="err_tol" value="$(arg err_tol)"/>
	<param name="traj_id" value="$(arg traj_drone2)"/>
	</node>
   </group>

   <group ns="drone3">
   	<node pkg="bridge_px4" type="sp_pos_e_node" name ="sp_pos_e_node" output="screen">
        <param name="t_fs" value="$(arg t_fs)"/>
	<param name="err_tol" value="$(arg err_tol)"/>
	<param name="traj_id" value="$(arg traj_drone3)"/>
	</node>
   </group>
</launch>
